timeless/access_control/views.py

```diff
@@ -17,4 +17,4 @@ def get(self, company_id):
     def dispatch_request(self, *args, **kwargs):
         method = getattr(self, request.method.lower(), None)
         assert method is not None, "Unimplemented method %r" % request.method
-        super(SecuredView, self).dispatch_request(self, args, kwargs)
+        super(SecuredView, self).dispatch_request(kwargs)
```

Comment:
@amihaiemil Was there a reason to pass args such way? Correct way should be like:
```python
super(SecuredView, self).dispatch_request(*args, **kwargs)
```